client/src/app/pages/archetypes/components/archetype-detail-drawer.tsx

```
@@ -119,7 +121,7 @@ const ArchetypeDetailDrawer: React.FC<IArchetypeDetailDrawerProps> = ({
                   {t("terms.tagsCriteria")}
                 </DescriptionListTerm>
                 <DescriptionListDescription>
-                  {archetype?.criteria?.length ?? 0 > 0 ? (
+                  {(archetype?.criteria?.length ?? 0 > 0) ? (
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)